src/addgrid.f

```diff
@@ -1,67 +1,47 @@
+!>    @file
+!>    @brief This subroutine packs up a Grid Definition Section
+!>    (Section 3) and adds it to a GRIB2 message.
+!>    @author Gilbert ORG: W/NP11 @date 2000-05-01
+!>
```

Comment:
Need to doocumentation blocks.

Basically, end the first one, then a blank line, then start a new one.

Reply:
addressed, Thanks!

Reply:
addressed, Thanks!